app/forms/link_case/confirmation_form.rb

```diff
@@ -0,0 +1,16 @@
+module LinkCase
+  class ConfirmationForm < BaseForm
+    form_for LinkedApplication
+
+    attr_accessor :link_type_code
+
+    validates :link_type_code, presence: true, unless: proc { draft? }
+
+    def save
+      return if invalid? || draft?
+
+      model.update!(link_type_code:)
```

Comment:
I think we need to update the legal_aid_application in here as well since we jump over the question "Do you want to link the case?" when they have copied so it is not set. This would require retrieving the legal_aid_application, if not already available, then below should do it

```ruby
  legal_aid_application.update!(link_case: link_type_code.present?)
```
tests need to cover this too if we do this

Reply:
changes made in pairing session